### PR TITLE
Replace missing static text with '<factor header value> not specified' if factor value is empty

### PR DESCRIPTION
**Pull request by upendrakumbham**
Replace missing static text with '<cell line> not specified' or '<disease> not specified' if the corresponding factor value is empty.

### CODE REVIEW

File: src/tabs/heatmap/column-filters/ColumnFiltersSection.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const headerName = (name) => (
   + `: `
 )
 
+const humanize = (name)=> (
+	name.split(`_`).join(` `).toLowerCase()
+)
+
```
Comment (alfonsomunozpomer) [inline src/tabs/heatmap/column-filters/ColumnFiltersSection.js:18]: For the record, if `name` is the only argument in the function and the result is the return value of `toLowerCase` you can simplify this to:
```javascript
const humanize = name => name.split(`_`).join(` `).toLowerCase()
```

And going one step further, notice that this `split` and `join` is equivalent to:
```javascript
const humanize = name => name.replaceAll(`_`, ` `).toLowerCase()
```

**Review by alfonsomunozpomer — COMMENTED** [review]
I’m not entirely clear where `name` comes from in lines 118 in `ColumnFilterSection.js`. Have you tested this case?

File: src/tabs/heatmap/column-filters/ColumnFiltersSection.js (hunk 1 above)
Comment (upendrakumbham) [reply src/tabs/heatmap/column-filters/ColumnFiltersSection.js]: Replaced with above. Thanks!

Comment (upendrakumbham) [reply src/tabs/heatmap/column-filters/ColumnFiltersSection.js]: Do I need to publish this change

**Comment by upendrakumbham** [comment]
> Also, be careful about the indentation settings in your IDE, since changing the indentation just introduces unnecessary noise that reviewers need to filter out.

Hi @alfonsomunozpomer, I tried, but I couldn't manage to fix these indentation issues. I need your help to fix this.

**Review by alfonsomunozpomer — APPROVED** [review]
All good now.

**Pull request merged**